go/vm/evmc/evmc_interpreter.go

```diff
@@ -155,8 +155,9 @@ func toEvmcRevision(revision vm.Revision) (evmc.Revision, error) {
 // context and chain state information external to the the interpreter.
 // It implements the host interface of evmc's Go bindings.
 type hostContext struct {
-	params  vm.Parameters
-	context vm.RunContext
+	params         vm.Parameters
+	context        vm.RunContext
+	evmcBlobHashes []evmc.Hash
```

Comment:
I don't think this interface is clean. I understand where does this come from, but I don't think we should be mixing vm and evmc types like this.